### PR TITLE
Renaming pandas to pandas_magic to avoid circular import problem

### PR DESCRIPTION
**Pull request by charlie-sanders**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +56,7 @@ In [3]: %%prql
    ...:   aggregate [average unitPrice]
    ...: )
 In [4]: %%prql results <<
-   ...: from p = products.csv
+   ...: from p
```
Comment (max-sixty) [inline README.md:59]: Does this work? Is there a table called `p`? I realize it's that way above (which maybe I wrote!)..

File: pyprql/pandas_magic/__init__.py [hunk 2]
```diff
@@ -9,4 +9,4 @@
 results_df = df.prql.query('from df | select [age,name,occupation] | filter age > 21')
 
 """
-import pyprql.pandas.prql  # noqa: F401
+import pyprql.pandas_magic.prql  # noqa: F401
```
Comment (max-sixty) [inline pyprql/pandas_magic/__init__.py:12]: FYI if you come back to this, I would might call it `pandas_accessor` or similar, `magic` is the technical name for the `%%` functionality in jupyter / ipython. But no biggie

Comment (charlie-sanders) [reply pyprql/pandas_magic/__init__.py:12]: Ahh , will do.

File: README.md (hunk 1 above)
Comment (charlie-sanders) [reply README.md:59]: if 'p' is a table in the database yes, I reworked the %%prql magic docs to use a full database connection string, if using CSV I think we should push the pandas accessor.

**Pull request merged**